VacaYAY/DataAccesLayer/Data/ApplicationDbContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿using DataAccesLayer.Configuration;
+using DataAccesLayer.Entities;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+
+namespace DataAccesLayer.Data
+{
+    public class ApplicationDbContext : IdentityDbContext<Employee, IdentityRole<int>, int>
+    {
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options) { }
+
+        protected override void OnModelCreating(ModelBuilder builder)
```

Comment:
Let's set some rules about for code order:

1. DbSets
2. Methods 

It would make code much easier to read